src/actionCreators/messages.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import { clearingResourceURIMessage } from 'actions/index'
+
+const clearResourceURIMessage = () => (dispatch, getState) => {
```

Comment:
Why is there an actionCreator? The action isn't asynchronous.

Reply:
I think I can simplify this into just the template. I was following the pattern used in `loadLanguages` but it seems unnecessary.